### PR TITLE
allow a proxy_server

### PR DESCRIPTION
**Pull request by wdpypere**
I am no ruby expert but something like this should resolve https://github.com/OSC/ondemand/issues/73

### CODE REVIEW

File: ood-portal-generator/lib/ood_portal_generator/view.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ def initialize(opts = {})
       @ssl              = opts.fetch(:ssl, nil)
       @listen_addr_port = opts.fetch(:listen_addr_port, nil)
       @servername       = opts.fetch(:servername, nil)
+      proxy_server      = opts.fetch(:proxy_server, @servername)
```
Comment (treydock) [inline ood-portal-generator/lib/ood_portal_generator/view.rb]: This should be `@proxy_server`. Missing `@` to denote instance variable.

**Comment by treydock** [comment]
You'll also want to update `ood-portal-generator/share/ood_portal_example.yml`. I'd recommend making note the default is value for `servername`.

**Comment by wdpypere** [comment]
@treydock OK, I'll have a look at the example

**Comment by wdpypere** [comment]
The tests fail on CentOS 7 because it can't find rake. is this expected?

**Comment by treydock** [comment]
It's a known issue. The tests install `ondemand-runtime` and normally that is version 1.6 but I've pushed 1.7 for testing which isn't compatible with the tests.

**Comment by johrstrom** [comment]
Is there anything holding this up?  @ericfranz we want this as a part of 6-18?

**Pull request merged**